SKYKitChat/Classes/Core/SKYChatExtension.h

```diff
@@ -194,9 +194,11 @@ typedef void (^SKYChatConversationCompletion)(SKYConversation *_Nullable convers
 
  @param completion completion block
  */
-- (void)fetchUserConversationsWithCompletion:
-    (SKYChatFetchUserConversationListCompletion _Nullable)completion
-    NS_SWIFT_NAME(fetchUserConversations(completion:));
+- (void)fetchUserConversationsWithFetchLastMessage:(BOOL)fetchLastMessage
+                                        completion:
+                                            (SKYChatFetchUserConversationListCompletion _Nullable)
+                                                completion
+    NS_SWIFT_NAME(fetchUserConversations(fetchLastMessage:completion:));
```

Comment:
It will be better to keep `fetchUserConversationsWithCompletion:` with auto fill the fetchLastMessage as `YES` and call this `fetchUserConversationsWithFetchLastMessage`.

It will maintain backward compatibility. And also provide sensible default.